src/main/java/net/mcreator/element/ModElementTypeLoader.java

```diff
@@ -50,6 +50,8 @@ public static void loadModElements() {
 		ModElementType.ENCHANTMENT = register(
 				new ModElementType<>("enchantment", 'm', BaseType.OTHER, RecipeType.NONE, EnchantmentGUI::new,
 						Enchantment.class));
+		ModElementType.FEATURE = register(
```

Comment:
As @Goldorion suggested, f should be used for features instead of fluids I think, as average workspace will probably have more features than fluids